components/script/dom/attr.rs

```diff
@@ -291,33 +148,41 @@ impl AttrMethods for Attr {
 impl Attr {
     pub fn set_value(&self, mut value: AttrValue, owner: &Element) {
         assert!(Some(owner) == self.owner().r());
+        let notify = self.name.namespace == ns!("");
+        if notify {
+            owner.will_mutate_attr();
```

Comment:
Probably, yeah. I was trying to mimic the existing behavior here, but the existing behavior is probably just wrong.